src/main/java/hudson/plugins/plot/PlotPublisher.java

```diff
@@ -1,166 +0,0 @@
-/*
- * Copyright (c) 2007-2009 Yahoo! Inc.  All rights reserved.
- * The copyrights to the contents of this file are licensed under the MIT License (http://www.opensource.org/licenses/mit-license.php)
- */
-package hudson.plugins.plot;
-
-import hudson.Launcher;
-import hudson.model.Action;
-import hudson.model.BuildListener;
-import hudson.model.AbstractBuild;
-import hudson.model.AbstractProject;
-import hudson.tasks.BuildStepDescriptor;
-import hudson.tasks.Publisher;
-
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.logging.Logger;
-
-import org.apache.commons.collections.CollectionUtils;
-
-/**
- * Records the plot data for builds.
- *
- * @author Nigel Daley
- */
-public class PlotPublisher extends AbstractPlotPublisher {
```

Comment:
🐛 I think this probably will be a problem when loading publishers created with older versions of the plugin, since the Publisher class has now changed -- also will be a problem if any plugins depend on this one, because they will be expecting classes to exist that are removed as of this change.